imageroot/events/smarthost-changed/10handler

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+${AGENT_INSTALL_DIR}/bin/expand-smarthost && systemctl restart "${MODULE_ID}.service"
```

Comment:
I was wondering: can we add the expand-smarthost as `ExecStartPre` inside the systemd unit? So you will not need to execute it manually.